### PR TITLE
task/8387/edit profile user

### PR DESCRIPTION
**Pull request by KhaiCM**
https://edu-redmine.framgia.vn/issues/8387

### CODE REVIEW

**Comment by KhaiCM** [comment]
ready

File: public/backend/css/style.css [hunk 1]
```diff
@@ -239,20 +298,25 @@ tr:nth-child(even) {
...
     float: left;
     padding: 8px 16px;
     text-decoration: none;
 }
+
 .pagination li.active {
     background-color: #4CAF50;
     color: white;
-}
+}
```
Comment (BunnyPi04) [inline public/backend/css/style.css]: Thiếu  dòng trắng cuối file

File: resources/views/backend/users/detail.blade.php [hunk 2]
```diff
@@ -2,45 +2,54 @@
...
+    </div>
+</div>
+<div class="form-group row">
+    <div class="col-sm-10">
+        {!! Form::submit(__('label.changes'), ['class' => 'btn btn-primary float-right']) !!}
+    </div>
+</div>
 {!! Form::close() !!}
-@endsection
+@endsection
```
Comment (BunnyPi04) [inline resources/views/backend/users/detail.blade.php]: thiếu dòng trắng cuối file

File: app/Http/Controllers/Admin/UserController.php [hunk 3]
```diff
@@ -88,16 +86,29 @@ public function edit($id)
...
-            return redirect(route('user.detail', $id))->with('message', __('label.edit_sussess'));
-        }
-        catch (ModelNotFoundException $ex) 
-        {
-            echo $ex->getMessage();
+            if (file_exists($user->avatar)) {
+                unlink(config('app.avatar_path') . $user->avatar);
+            }
+            if ($user->link != null) {
+                $user->avatar =null;
```
Comment (buihanh2304) [inline app/Http/Controllers/Admin/UserController.php]: convention

File: app/Http/Controllers/UserPageController.php [hunk 4]
```diff
@@ -85,17 +102,29 @@ public function edit($id)
...
-            return redirect(route('user_page.edit',$id))->with('message', __('label.edit_sussess'));
-        }
-        catch (ModelNotFoundException $ex) 
-        {
-            echo $ex->getMessage();
+            if (file_exists($user->avatar)) {
+                unlink(config('app.avatar_path') . $user->avatar);
+            }
+            if ($user->link != null) {
+                $user->avatar =null;
```
Comment (buihanh2304) [inline app/Http/Controllers/UserPageController.php]: convention

**Pull request merged**